scripts/generate_road_colours.py

```diff
@@ -1,47 +1,17 @@
 #!/usr/bin/env python2
-
 from colormath.color_conversions import convert_color
 from colormath.color_objects import LabColor, LCHabColor, sRGBColor
 from colormath.color_diff import delta_e_cie2000
 import numpy
-
 from collections import OrderedDict, namedtuple
 
-class Color:
-  def __init__(self, lch_tuple):
-    self.m_lch = LCHabColor(*lch_tuple)
-
-  def lch(self):
-    return "Lch({:.0f},{:.0f},{:.0f})".format(*(self.m_lch.get_value_tuple()))
-
-  def rgb(self):
-    rgb = convert_color(self.m_lch, sRGBColor)
-    if (rgb.rgb_r != rgb.clamped_rgb_r or rgb.rgb_g != rgb.clamped_rgb_g or rgb.rgb_b != rgb.clamped_rgb_b):
-      raise Exception("Colour {} is outside sRGB".format(self.lch()))
-    return rgb.get_rgb_hex()
-
-  def rgb_error(self):
-    return delta_e_cie2000(convert_color(self.m_lch, LabColor),
-                           convert_color(sRGBColor.new_from_rgb_hex(self.rgb()), LabColor))
 
 road_classes = ["motorway", "trunk", "primary", "secondary"]
-colour_divisions = len(road_classes) - 1
-hues = OrderedDict()
 
 # The minimum and maximum hue for the road colours
-# Because hue is a circle, it may be needed to add/subtract 360 to the min or
-# max when changing them
```

Comment:
This comment should go with where `min_h` and `max_h` is set. Unless it's not true - I never did test to see if it was necessary. To get between two colours on a circle you can go the long way or the short way.